cypress/integration/ta/queue_check_in_check_out.spec.js

```diff
@@ -165,7 +167,7 @@ describe("Checking in and out when there arent scheduled office hours, but the o
         });
     });
   });
-
+/*
   it("checking in a TA when there is not scheduled office hours", function () {
```

Comment:
I think you should be able to uncomment this test now - We probably want to test that queues with no one checked in don't show up as queue cards.

Reply:
deleted test to open queue clean from today page since they are hidden if closed